src/excmds.ts

```diff
@@ -3872,6 +3873,27 @@ export async function jsb(...str: string[]) {
     }
 }
 
+//#background_helper
+import * as Parser from "rss-parser"
+import * as semverCompare from "semver-compare"
+
+//#background
```

Comment:
I should double check that the RSS parser works in the background, as I initially had this as content until fillcmdline complained.

Reply:
It even does work.